style.css

```diff
@@ -0,0 +1,36 @@
+#main {
+  /* margin: 0 auto; */
+
+  /* text-align: center; */
+  background-color: #3333;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+#img-div {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  justify-content: center;
+  align-items: center;
+}
+#image {
+  max-width: 100%;
+  height: auto;
+}
+#tribute-info {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  /* align-items: center; */
+  /* text-align: center; */
```

Comment:
- [ ] Please remove commented lines.

Reply:
I think it would be good to give a reason why commented lines should be removed, as this would give more context to the change requested.